features/steps/search_api.go

```diff
@@ -0,0 +1,29 @@
+package steps
```

Comment:
This can be moved to `dp-component-test` as `FakeAPIClient`

Reply:
True but that can be done in a different PR